README.NL.rst

```diff
@@ -0,0 +1,53 @@
+=====================
+Record Management App
+=====================
+
+:Version: 0.1.0
+:Source: https://github.com/GemeenteUtrecht/record-management-app
+:Keywords: Common Ground, Record Management, Archiveren
+:PythonVersion: 3.8
+
+Zaakgericht record-management in het `Common Ground`_ landschap.
+
+Ontwikkeld door `Maykin Media B.V.`_ voor Gemeente Utrecht en Gemeente Delft.
+
+
+Introductie
+===========
+
+Archiefbeheer draait om het vernietigen van gegevens die aan het einde van hun levensduur zijn gekomen.
+Zaakgericht werken schrijft de archiveringsvoorwaarden voor, voor zaken die zijn beëindigd.
+
+De Record Management App (RMA) biedt functionaliteit voor archiefbeheer om de vernietiging van zaken in te plannen
+volgens de archiveringsvoorwaarden. De RMA volgt een proces dat uit meerdere stappen bestaat, over verschillende rollen.
+
+Er is aandacht besteed aan de gebruikerservaring van de beheerders die de app gebruiken, met notificaties
+om te informeren over lopende taken, volledige auditregistratie en traceerbaarheid van gebruiker- en
+systeemacties.
+
+De RMA ondersteunt alle backends die de 1.0.x `API's voor Zaakgericht Werken`_ implementeren.
+
```

Comment:
I think quick-start section should also be here in case dutch speakers won't take a look at english README